docs/standard/base-types/how-to-pad-a-number-with-leading-zeros.md

```diff
@@ -38,19 +39,19 @@ The following example formats several integer values with leading zeros so that
 
 1. Determine how many leading zeros you want the integer value to display.
 
-1. Determine whether you want to display the integer as a decimal value or a hexadecimal value.
+1. Determine whether you want to display the integer as a decimal or a hexadecimal value.
 
-    - Formatting it as a decimal value requires that you use the "D" standard format specifier.
+    - Formatting it as a decimal value requires the "D" standard format specifier.
 
-    - Formatting it as a hexadecimal value requires that you use the "X" standard format specifier.
+    - Formatting it as a hexadecimal value requires the "X" standard format specifier.
 
 1. Determine the length of the unpadded numeric string by calling the integer value's `ToString("D").Length` or `ToString("X").Length` method.
 
-1. Add the number of leading zeros that you want to include in the formatted string to the length of the unpadded numeric string. Adding the number of leading zeros defines the total length of the padded string.
+1. Add the number of leading zeros you want to include in the formatted string to the length of the unpadded numeric string. Adding the number of leading zeros defines the total length of the padded string.
```

Comment:
I think removing `that` makes the sentence less clear, it should be rewritten or `that` added back.
Maybe something like this?

```suggestion
1. Add the number of leading zeroes you want to include in the formatted string. The number of leading zeros defines the total length of the padded string.
```
@tdykstra what do you think?

Reply:
I would both add back "that" and rewrite, something like this:

```suggestion
1. Add the number of leading zeros to the length of the unpadded numeric string. The result is the total length of the padded string.
```